apps/backoffice-v2/src/domains/workflows/fetchers.ts

```diff
@@ -214,6 +214,7 @@ export const fetchWorkflowById = async ({
   filterId: string;
 }) => {
   const [workflow, error] = await apiClient({
+    // TODO: filterId supposedly doesn't do anything on the backend in this endpoint
```

Comment:
It applies select/include. Part of my task is to remove the use of filter form the endpoint.